Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -293,12 +295,14 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
 
   // Request an XR session on the user device with WebXR,
   // typically in response to a user request such as a button press
-  publicAPI.startXR = (isAR) => {
+  publicAPI.startXR = (xrSessionType) => {
```

Comment:
This should be added to the openglrenderwindow type.

Reply:
Added